docs/content/config/advanced/tips/server-multiple-ip-addresses.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Advanced | Tips | Server with multiple IP-addresses'
+---
+
+## Server with multiple IP-addresses
```

Comment:
```suggestion
---
title: 'Use Cases | Binding DMS to a specific network'
hide:
  - toc
---
```

No need to repeat the title :)

Also added extra metadata to disable the right-side navigation menu as that's not needed for this page.